src/web/mjs/connectors/Caisemh.mjs

```diff
@@ -0,0 +1,24 @@
+import MH from './templates/MH.mjs';
+
+export default class Caisemh extends MH {
+
+    constructor() {
+        super();
+        super.id = 'caisemh';
+        super.label = 'Caisemh';
+        this.tags = [ 'manga', 'webtoon', 'chinese', 'hentai' ];
+        this.url = 'https://www.caisemh.com';
+
+        this.queryPages = 'div#cp_img source';
+    }
+
+    async _getPages(chapter) {
+        const request = new Request(new URL(chapter.id, this.url), {
+            headers: {
+                'x-user-agent': 'Mozilla/5.0 Android X'
```

Comment:
Does this work with this user agent?
I was just suggesting with some random string.

Reply:
it works fine, but il change it back to the useragent that i had before

Reply:
It is possible that the server just checked if the user agent contains `ios|android`, so it probably worked with this minimalistic user agent